docs/getting-started/dockstore-tools.rst

```diff
@@ -1,18 +1,30 @@
-.. note::
-  This tutorial is a continuation of :doc:`Creating a Dockstore Account </getting-started/register-on-dockstore>`. Please complete
-  the tutorial prior to doing this one.
-
 Register a Tool on Dockstore
 ############################
 
+.. contents::
+   :local:
+   :depth: 2
+
 Tutorial Goals
 ==============
 
 -  Discover how to register a tool on Dockstore
 -  Publish your tool
 
+Assumptions
+-----------
+This document assumes you have already have a tool ready to register. You do not need to have a Dockerfile associated with it unless you are using one our legacy registration methods (more on that below).
+
+If you followed :doc:`Getting Started With CWL </getting-started/getting-started-with-cwl>` to create a CWL tool, you will now have your ``Dockerfile`` and ``Dockstore.cwl`` in GitHub, have setup Quay.io to automatically build your Docker image, and have linked your accounts to Dockstore. This will allow you to use any of our registration methods, including legacy methods. Of course, you can always follow along with your own unique tools too, which may or may not be associated with an autobuilding Docker image.
+
+Regardless of how you made your tool, this tutorial will assume you are using CWL. Galaxy and Nextflow do not have a concept of tools, so if you are workfing in those languages, :doc:`please see our documentation on workflows instead </getting-started/dockstore-workflows>`.
+
+.. note:: The WDL community does not explicitly differentiate :doc:`tools versus workflows </getting-started/intro-to-dockstore-tools-and-workflows>`. However, Dockstore allows the registration of "WDL tools" as a legacy feature using legacy registration methods. We encourage people to :doc:`register WDLs as workflows </getting-started/dockstore-workflows>`. The GitHub App supports the registration of WDL workflows, but it does not support registration of "WDL tools."
```

Comment:
Suggest more succinct language: 

The WDL community does not explicitly differentiate :doc:`tools versus workflows </getting-started/intro-to-dockstore-tools-and-workflows>`. However, Dockstore allows the registration of "WDL tools" as a legacy or GitHub App registration methods, but we encourage people to :doc:`register WDLs as workflows </getting-started/dockstore-workflows>`.